reasult

eye-opener-tenzin opened the pull request.

eye-opener-tenzin commented:
Result component

Zyanvel reviewed (changes requested):
I observe that in pages/index.js, on line 2, an Image was deleted, is this image required for deploying the app?

Also, under utils/requests.js on line 1, the previous API KEY was deleted, may I ask why?

Thank you, great job!

eye-opener-tenzin commented:
It is not using so I deleted it, API_KEY is still there but the environment
folder is deleted I am rendering API_KEY directly in the request folder.
.env.local always showing error when rendering onto webpage.

On Mon, 2 Aug 2021 at 09:29, Zyanvel ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> I observe that in pages/index.js, on line 2, an Image was deleted, is this
> image required for deploying the app?
>
> Also, under utils/requests.js on line 1, the previous API KEY was deleted,
> may I ask why?
>
> Thank you, great job!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Zyanvel/huluu/pull/3#pullrequestreview-720390193>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLA6PP3W7FBJXIB66FZZ63T23BXTANCNFSM5BMJMMIQ>
> .
>

Zyanvel commented:
> It is not using so I deleted it, API_KEY is still there but the environment folder is deleted I am rendering API_KEY directly in the request folder. .env.local always showing error when rendering onto webpage.
> […](#)
> On Mon, 2 Aug 2021 at 09:29, Zyanvel ***@***.***> wrote: ***@***.**** requested changes on this pull request. I observe that in pages/index.js, on line 2, an Image was deleted, is this image required for deploying the app? Also, under utils/requests.js on line 1, the previous API KEY was deleted, may I ask why? Thank you, great job! — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <[#3 (review)](https://github.com/Zyanvel/huluu/pull/3#pullrequestreview-720390193)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALLA6PP3W7FBJXIB66FZZ63T23BXTANCNFSM5BMJMMIQ> .

I see, thank you. What about the image?

eye-opener-tenzin commented:
there is no any image there, i just deleted the second line where it is
never using.

On Mon, 2 Aug 2021 at 09:46, Zyanvel ***@***.***> wrote:

> It is not using so I deleted it, API_KEY is still there but the
> environment folder is deleted I am rendering API_KEY directly in the
> request folder. .env.local always showing error when rendering onto webpage.
> … <#m_5161453937759614080_>
> On Mon, 2 Aug 2021 at 09:29, Zyanvel *@*.*> wrote: @.** requested changes
> on this pull request. I observe that in pages/index.js, on line 2, an Image
> was deleted, is this image required for deploying the app? Also, under
> utils/requests.js on line 1, the previous API KEY was deleted, may I ask
> why? Thank you, great job! — You are receiving this because you authored
> the thread. Reply to this email directly, view it on GitHub <#3 (review)
> <https://github.com/Zyanvel/huluu/pull/3#pullrequestreview-720390193>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/ALLA6PP3W7FBJXIB66FZZ63T23BXTANCNFSM5BMJMMIQ
> .
>
> I see, thank you. What about the image?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Zyanvel/huluu/pull/3#issuecomment-891173591>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLA6PIHMTRO4TCS3FVHCXDT23DXHANCNFSM5BMJMMIQ>
> .
>

Zyanvel commented:
Perfect, thank you.

eye-opener-tenzin commented:
API_KEY is confidential and it is from my account please don't share git on
public.

On Mon, 2 Aug 2021 at 09:40, Tenzin-eye-opener ***@***.***> wrote:

> It is not using so I deleted it, API_KEY is still there but the
> environment folder is deleted I am rendering API_KEY directly in the
> request folder.  .env.local always showing error when rendering onto
> webpage.
>
> On Mon, 2 Aug 2021 at 09:29, Zyanvel ***@***.***> wrote:
>
>> ***@***.**** requested changes on this pull request.
>>
>> I observe that in pages/index.js, on line 2, an Image was deleted, is
>> this image required for deploying the app?
>>
>> Also, under utils/requests.js on line 1, the previous API KEY was
>> deleted, may I ask why?
>>
>> Thank you, great job!
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Zyanvel/huluu/pull/3#pullrequestreview-720390193>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ALLA6PP3W7FBJXIB66FZZ63T23BXTANCNFSM5BMJMMIQ>
>> .
>>
>

spagnon7 reviewed (approved):
Looks really great to me, nice job!
